source/agora/node/FullNode.d

```diff
@@ -545,11 +545,14 @@ public class FullNode : API
             this.startTaskTimer(TimersIdx.BlockCatchup, this.config.node.block_catchup_interval);
         }
 
-        if (this.network.peers.empty())  // no clients yet (discovery)
+        if (this.network.peers.empty())
+        {
+            // TODO: Iff the node is the single validator for the network,
```

Comment:
```suggestion
            // TODO: If the node is the single validator for the network,
```

Reply:
If it is the only validator then the message is still correct.

Reply:
`Iff` is correct isn't it ?
Also the message is correct, but it should not be a warning. It shouldn't even be logged IMO.

Reply:
ok so `Iff` as in `If and only if`